src/main/java/org/quiltmc/loader/impl/metadata/BuiltinModMetadata.java

```diff
@@ -46,14 +46,25 @@
 	private final ContactInformation contact;
 	private final Collection<String> license;
 	private final NavigableMap<Integer, String> icons;
+	private final Collection<ModDependency> depends;
+	private final Collection<ModDependency> recommends;
+	private final Collection<ModDependency> suggests;
+	private final Collection<ModDependency> conflicts;
+	private final Collection<ModDependency> breaks;
 
 	private BuiltinModMetadata(String id, Version version,
 			ModEnvironment environment,
 			String name, String description,
 			Collection<Person> authors, Collection<Person> contributors,
 			ContactInformation contact,
 			Collection<String> license,
-			NavigableMap<Integer, String> icons) {
+			NavigableMap<Integer, String> icons,
```

Comment:
All of these need to be made unmodifiable. Mod metadata should be immutable and upstream got lazy